src/main/kotlin/com/criptext/mail/scenes/signin/ForgotPasswordDialog.kt

```diff
@@ -0,0 +1,67 @@
+package com.criptext.mail.scenes.signin
+
+import android.content.Context
+import android.support.v4.content.ContextCompat
+import android.support.v7.app.AlertDialog
+import android.support.v7.app.AppCompatActivity
+import android.view.Gravity
+import android.view.View
+import android.widget.Button
+import android.widget.LinearLayout
+import android.widget.TextView
+import com.criptext.mail.R
+import com.criptext.mail.utils.EmailAddressUtils
+
+/**
+ * Created by sebas on 3/8/18.
+ */
+
+class ForgotPasswordDialog(val context: Context) {
+    private var dialog: AlertDialog? = null
+    private val res = context.resources
+    private var emailAddress: String = ""
```

Comment:
Why not pass this in the constructor? There is nor reason for this to be mutable.